docs/api-docs/getting-started/rest-api-authentication.md

```diff
@@ -87,6 +87,7 @@ Where both types of API account are supported, review the preceding sections to
 | API or Use Case | App API Account | Store API Account |
 |:----------------|:---------------:|:-----------------:|
 | [REST Store Management APIs](/api-docs/getting-started/authentication#access-tokens) | &times; | &times; |
+| [GraphQL Store Management APIs](/api-docs/getting-started/authentication#access-tokens) | &times; | &times; |
```

Comment:
@solofeed Do you know if this GraphQL Data Layer API works with App API Accounts (or if you can only authenticate via Store API Accounts)?

Reply:
k, i'll leave Store API Account because I'm confident about that one. @slsriehl not sure if my table is misleading, since we don't have that many GraphQL Management APIs GA yet